src/zope/testrunner/tb_format.py

```diff
@@ -20,9 +20,34 @@
 import zope.testrunner.feature
 
 
+try:
+    _iter_chain = traceback._iter_chain
+except AttributeError:
+    # Python 3.5
```

Comment:
BTW Python 2.7 doesn't have `traceback._iter_chain` either, but I suppose there are no chained exceptions on Python 2, so that never came up.